web-services/model/src/main/java/datawave/webservice/query/model/ModelBean.java

```diff
@@ -127,6 +131,32 @@ public class ModelBean {
     @GZIP
     @Interceptors(ResponseInterceptor.class)
     public ModelList listModelNames(@QueryParam("modelTableName") String modelTableName) {
+        return listModelNames(modelTableName, false);
```

Comment:
If the default for skipCache is true down below, should this not be passing in true?  Moreover do we even need this method since the query parameter in the method below is basically optional?

Reply:
Since this endpoint is open to everyone, my assumption was that we wouldn't want a typical user to be able to circumvent the webservice's cache layer. So, the default of `true` below is only in effect for the admin-only override, just as a convenience. Same reasoning was applied to the `getModel` endpoint as well, since it's also open to everyone.

But if you think it's ok to relax that restriction in this case, then that would definitely simplify things. I could go either way here, honestly